0004/README.md

```diff
@@ -0,0 +1,416 @@
+# Proposal 4: SLSA Conformance Program
+* Proposers: [Kris K](https://github.com/kpk47) (kkris@google.com), [Joshua Mulliken](https://github.com/JoshuaMulliken) (jmullike@redhat.com)
+* GitHub issue: https://github.com/slsa-framework/slsa/issues/515
+* Status: [DRAFT](../README.md#meaning-of-status-codes)
+
+## Abstract
+
+This document describes a mechanism for labeling build platforms as officially
+SLSA conformant, the "SLSA Conformance Program." This proposal lays out the
+program at a high level, including a draft of the survey build platforms would
+need to fill out to participate. Please focus review on the program as a whole
+and the survey's general contents, reserving discussion of the survey's final
+wording for implementation PRs.
+
+## Problem statement
+
+One of SLSA's
+[guiding principles](https://slsa.dev/spec/v1.0/principles#trust-platforms-verify-artifacts)
+is to "Trust platforms, verify artifacts." The SLSA v1.0 specification reflects
+this principle's asymmetry through how it treats [verifying build
+platforms](https://slsa.dev/spec/v1.0/verifying-systems) and [verifying
+artifacts](https://slsa.dev/spec/v1.0/verifying-artifacts) – the former is a
+list of freeform questions a verifier might wish to ask of a build system, and
+the latter is a detailed procedure to be implemented in a tool. It is difficult
+and expensive to answer the question "is this build platform SLSA conformant?"  
+
+As a practical matter, the OpenSSF controls SLSA and could be in a position to
+determine which build platforms are SLSA conformant. However, the OpenSSF
+doesn't have the desire or resources to do so. Furthermore, the SLSA
+specification makes it the verifier's responsibility to determine whether a
+build system is trustworthy, and that trustworthiness determines Build L3
+conformance. Instead, the OpenSSF should do what it does best: build a
+community. In this case, that community would consist of build platforms,
+verifiers, and independent auditors who hold each other accountable for build
+platform security.   
+
+The OpenSSF's tool for building this community is a SLSA Conformance Program.
+This document presents high-level objectives for the SLSA Conformance Program
+and a design for the program based loosely on the CNCF's Certified Kubernetes
+Conformance Program.
+
+## Requirements
+
+Required:
+
+1. Set a baseline for Build L3 build platforms
+1. Bootstrap an ecosystem/market of build platforms and auditors
+1. Create a sensible process for managing SLSA trademarks
+1. Limit or mitigate perverse incentives for auditors
+1. Scale from new market entrants to existing large providers
+
+Nice to have:
+
+1. Distribute provenance verification materials
```

Comment:
How would this alone answer whether a build platform is v1.0 L3-compliant? If it cannot, why is this here?